packages/manager/src/features/VPCs/VPCCreate/FormComponents/VPCTopSectionContent.tsx

```diff
@@ -109,6 +149,127 @@ export const VPCTopSectionContent = (props: Props) => {
           />
         )}
       />
+      {isDualStackEnabled && (
+        <Box marginTop={2}>
+          <FormLabel>Networking IP Stack </FormLabel>
+          <RadioGroup sx={{ display: 'block' }}>
+            <Grid container spacing={2}>
+              <SelectionCard
+                checked={!isDualStackSelected}
+                gridSize={{
+                  md: 3,
+                  sm: 12,
+                  xs: 12,
+                }}
+                heading="IPv4"
+                onClick={() => {
+                  remove(0);
+                }}
+                renderIcon={() => <Radio checked={!isDualStackSelected} />}
+                renderVariant={() => (
+                  <TooltipIcon
+                    status="info"
+                    sxTooltipIcon={{
+                      padding: '8px',
+                    }}
+                    text={
+                      <Typography>
+                        The VPC uses IPv4 addresses only. The VPC can use the
+                        entire RFC 1918 specified range for subnetting.
+                      </Typography>
+                    }
+                    width={250}
+                  />
+                )}
+                subheadings={[]}
+                sxCardBase={{ gap: 0 }}
+                sxCardBaseIcon={{ svg: { fontSize: '20px' } }}
+              />
+              <SelectionCard
+                checked={isDualStackSelected}
+                gridSize={{
+                  md: 3,
+                  sm: 12,
+                  xs: 12,
+                }}
+                heading="IPv4 + IPv6 (Dual Stack)"
+                onClick={() => {
+                  if (fields.length === 0) {
+                    append({
+                      range: '/52',
+                    });
+                  }
+                }}
+                renderIcon={() => <Radio checked={isDualStackSelected} />}
+                renderVariant={() => (
+                  <TooltipIcon
+                    status="info"
+                    sxTooltipIcon={{
+                      padding: '8px',
+                    }}
+                    text={
+                      <Stack spacing={2}>
+                        <Typography>
+                          The VPC supports both IPv4 and IPv6 addresses.
+                        </Typography>
+                        <Typography>
+                          For IPv4, the VPC can use the entire RFC 1918
+                          specified range for subnetting.
+                        </Typography>
+                        <Typography>
+                          For IPv6, the VPC is assigned an IPv6 prefix length of
+                          <Code>/52</Code> by default.
```

Comment:
```suggestion
                          For IPv6, the VPC is assigned an IPv6 prefix length of{' '}
                          <Code>/52</Code> by default.
```

Looks like we're missing a space here:

| this branch | suggestion |
|--------|--------|
| ![Screenshot 2025-07-10 at 9 35 25 AM](https://github.com/user-attachments/assets/52feeeac-40cc-4b26-acf7-7d2119096a94) | ![Screenshot 2025-07-10 at 9 36 27 AM](https://github.com/user-attachments/assets/665d087f-d0f0-48ea-be53-c36e29c672a3) |